en-af/jw300-baseline/readme.md

```diff
@@ -0,0 +1,22 @@
+# English to Afrikaans
+
+Author: Elan van Biljon
+
+## Data
+
+	- The JW300 English-Afrikaans dataset.
+
+## Model
+
+- Default Masakhane Transformer translation model.
+- [Link to google drive folder with models](https://drive.google.com/drive/folders/1GU683QZ0X22ItilcTtqAxVCd8Oc8s8g-?usp=sharing)
+
+## Analysis
+
+Smaller models perform drastically better due to better signal propagation dynamics.
```

Comment:
What kind of models did you try?

Reply:
I basically did the things I mentioned in the google group a few months back: (Use the default notebook transformer) reduce the depth of the model, reduce dropout. I want to explain more thoroughly but I'm actually trying to write that up for the ICLR AfricanNLP workshop. Would you mind if I retrospectively explain with that paper? :)

Reply:
That's cool! 😎